packages/one-app-bundler/package.json

```diff
@@ -53,7 +56,7 @@
     "enhanced-resolve": "^4.1.0",
     "eslint-loader": "^2.1.1",
     "expose-loader": "^0.7.5",
-    "fibers": "^4.0.1",
+    "fibers": "5.0.0",
```

Comment:
To support node 12 and above https://github.com/laverdet/node-fibers#supported-platforms